libraries/helpers.rb

```diff
@@ -36,7 +36,7 @@ def resource_attrs
 
 def pm2_is_app_online?(name)
   cmd = shell_out!('pm2 list', :returns => 0)
-  cmd.stdout =~ /#{name}/
+  not cmd.stdout.match(name).nil?
```

Comment:
Would you mind just changing 'not' with '!' so that Rubocop is happy? Thanks for this by the way.